docs/components/codemirror.vue

```diff
@@ -61,19 +62,19 @@ export default {
       default: false
     }
   },
-  data () {
+  data() {
```

Comment:
I wish prettier would get off their butts and add in the option/switch to allow the space here, as it makes searching for the function definition so much easier knowing that the def has the space, while usage does not.

Reply:
Yeah, I wish this would be configurable too.

Reply:
The entire point with prettier is using an optionized preset.

Reply:
Prettier has a discussion going on about adding a flag/option to set space between method name and the opening parenthesis (specifically when using object shorthand to define a method, but also for regular function definitions), and no space when calling a method.

Reply:
If they add in the option in a future release, we can always enable it to get the space back.